nntrainer/layers/acti_func.h

```diff
@@ -161,10 +161,10 @@ class ActiFunc {
    * @brief setActivation by custom activation function
    * @note  apply derivative as this activation_prime_fn does not utilize
    * derivative
-   * @param[in] std::function<Tensor(Tensor const &, Tensor &)> activation_fn
+   * @param[in] std::function<Tensor &(Tensor const &, Tensor &)> &activation_fn
```

Comment:
```
std::function<Tensor &(Tensor const &, Tensor &)> &
```
Is this information really needed? We can know it from definition.

Reply:
Removed it